packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# 48.18.0 - May 28, 2020
+
+## Fixed
+
+#### salesforcedx-vscode-lwc
+
+- Fix LWC preview on Windows when running `SFDX: Preview Component Locally` ([PR #2212](https://github.com/forcedotcom/salesforcedx-vscode/pull/2212), [Issue #1903](https://github.com/forcedotcom/salesforcedx-vscode/issues/1903))
+
+#### salesforcedx-utils-vscode
+
+- Correctly surface errors for failed CLI commands ([PR #2216](https://github.com/forcedotcom/salesforcedx-vscode/pull/2216), [Issue #2160](https://github.com/forcedotcom/salesforcedx-vscode/issues/2160), [Issue #2005](https://github.com/forcedotcom/salesforcedx-vscode/issues/2005))
```

Comment:
@rcoringrato-sfdc , consider rephrasing:
Display an error message in Apex Interactive Debugger when a user doesn’t have the required permission.

Reply:
@hbc-bindu @rcoringrato-sfdc the fix is not exclusive to errors caused by a user not having certain perms. We could rephrase it to `Display error messages in Apex Interactive Debugger ...`

Reply:
I take that back, the fix is not exclusive to the interactive debugger, that is just the place where we first noticed the problem. That code we fixed is used in other areas likes org authentication, isv debugger setup, etc.